generator/graphql-kotlin-schema-generator/src/test/kotlin/com/expediagroup/graphql/generator/internal/extensions/AnnotationExtensionsTest.kt

```diff
@@ -54,6 +55,14 @@ class AnnotationExtensionsTest {
     @GraphQLUnion(name = "MetaUnion", possibleTypes = [NoAnnotations::class])
     annotation class MetaUnion
 
+    @GraphQLDeprecated("class deprecated", ReplaceWith("WithAnnotations"))
+    @GraphQLIgnore
```

Comment:
is the ignore intentional?

Reply:
No, copy/pasted it from the original example. I removed it.